docs/en/user_guides/yolov5_tutorial.md

```diff
@@ -1 +1,249 @@
+```
+
+After executing the above command, the balloon dataset will be downloaded in the `data` folder with the converted format we need. The `train.json` and `val.json` are the annotation files, both are in the coco format.
+
+<div align=center>
+<img src="https://cdn.vansin.top/img/20220912105312.png" alt="image"/>
+</div>
+
+## Config File Preparation
+
+Create a new file called the `yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py` configuration file in the `configs/yolov5` folder, and copy the following content into it.
+
+```python
+_base_ = './yolov5_s-v61_syncbn_fast_8xb16-300e_coco.py'
+
+data_root = 'data/balloon/'
+
+train_batch_size_per_gpu = 4
+train_num_workers = 2
+
+metainfo = {
+    'CLASSES': ('balloon', ),
+    'PALETTE': [
+        (220, 20, 60),
+    ]
+}
+
+train_dataloader = dict(
+    batch_size=train_batch_size_per_gpu,
+    num_workers=train_num_workers,
+    dataset=dict(
+        data_root=data_root,
+        metainfo=metainfo,
+        data_prefix=dict(img='train/'),
+        ann_file='train.json'))
+
+val_dataloader = dict(
+    dataset=dict(
+        data_root=data_root,
+        metainfo=metainfo,
+        data_prefix=dict(img='val/'),
+        ann_file='val.json'))
+
+test_dataloader = val_dataloader
+
+val_evaluator = dict(ann_file=data_root + 'val.json')
+
+test_evaluator = val_evaluator
+
+model = dict(bbox_head=dict(head_module=dict(num_classes=1)))
+
+default_hooks = dict(logger=dict(interval=1))
+```
+
+The above configuration is inherited from `./yolov5_s-v61_syncbn_fast_8xb16-300e_coco.py`, and `data_root`, `metainfo`, `train_dataloader`, `val_dataloader`, `num_classes` and other configurations are updated according to the balloon data we are using.
+We set the `interval` of the logger to 1 because each iteration of the `interval` will output a loss-related log, and the balloon dataset we use is relatively tiny. We will not see the loss-related output if the `interval` is too large.
+
+## Training
+
+```shell
+python tools/train.py configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py
+```
+
+After executing the above training command, the `work_dirs/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon` folder will be automatically generated. Both the weight and the training configuration files will be saved in this folder.
+
+<div align=center>
+<img src="https://cdn.vansin.top/img/20220913213846.png" alt="image"/>
+</div>
+
+### Resume training after interruptions
+
+If training stops midway, add `--resume` at the end of the training command, and the program will automatically load the latest weight file from `work_dirs` to resume training.
+
+```shell
+python tools/train.py configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py --resume
+```
+
+### Fine-tune with loaded pretrained weights
+
+Our testing shows that with the YOLOv5-s pretrained model to train and validate, the coco/bbox_mAP can increase by more than 30 percent on the balloon dataset compared with not using the pre-training model solution.
```

Comment:
NOTICE: It is highly recommended that finetuning from large datasets, such as COCO, can significantly boost the performance of overall network. 
In this example, compared with training from scratch, finetuning the pretrained model outperforms with a significant margin. (Over 30+ mAP boost than training from scratch).